omod/src/test/java/org/openmrs/module/idgen/rest/resource/IdentifierSourceResourceTest.java

```diff
@@ -0,0 +1,68 @@
+/* * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.idgen.rest.resource;
+
+
+import org.openmrs.api.context.Context;
+import org.openmrs.module.idgen.IdentifierSource;
+import org.openmrs.module.idgen.rest.resource.IdentifierSourceResource;
+import org.openmrs.module.idgen.service.IdentifierSourceService;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.openmrs.module.webservices.rest.SimpleObject;
+import org.openmrs.module.webservices.rest.web.representation.Representation;
+import org.openmrs.module.webservices.rest.web.resource.impl.BaseDelegatingResourceTest;
+
+
+public class IdentifierSourceResourceTest extends BaseDelegatingResourceTest<IdentifierSourceResource, IdentifierSource> {
+	
+	public static final String IDENTIFIER_SOURCE_UUID = "0d47284f-9e9b-4a81-a88b-8bb42bc0a999";
+	
+	@Before
+	public void setup() throws Exception {
+		executeDataSet("org/openmrs/module/idgen/include/TestDataForLoadTest.xml");
+	}
+
+	@Override
+	public String getDisplayProperty() {
+		return "identifiersource";
+	}
+
+	@Override
+	public String getUuidProperty() {
+		return IDENTIFIER_SOURCE_UUID;
+	}
+
+	@Override
+	public IdentifierSource newObject() {
+		return Context.getService(IdentifierSourceService.class).getIdentifierSourceByUuid(getUuidProperty());
+	}
+	
+	
+	
+	@Override
+	public void validateDefaultRepresentation() throws Exception {
```

Comment:
Did you forget to add? super.validateDefaultRepresentation();

Reply:
added